app/pages/lab/workflow.cjsx

```diff
@@ -304,6 +304,22 @@ EditWorkflowPage = React.createClass
 
             </div>}
 
+          <div>
+            <div>
+              <AutoSave resource={@props.workflow}>
+                <span className="form-label">Pan and zoom</span><br />
+                <small className="form-help">Pan and zoom allows the user to zoom in and out and pan the subject in the subject viewer.</small>
```

Comment:
We use the term classification interface in the glossary that @BNewing is putting the finishing touches on, and we define the term so I think it would be good to use classification interface here